camp/camp-brooklyn/src/test/resources/visitors-creation-script.sql

```diff
@@ -19,18 +19,22 @@
 create database visitors;
 use visitors;
 
-# not necessary to create user if we grant (and not supported in some dialects)
-create user 'brooklyn' identified by 'br00k11n';
-
-grant usage on *.* to 'brooklyn'@'%' identified by 'br00k11n';
-
-# ''@localhost is sometimes set up, overriding brooklyn@'%', so do a second explicit grant
-grant usage on *.* to 'brooklyn'@'localhost' identified by 'br00k11n';
 
+# the below will create user (and note create user not supported in some dialects)
+grant usage on *.* to 'brooklyn'@'%' identified by '${config["creation.script.password"]}';
```

Comment:
Previously @aledsage deleted [this version](https://github.com/apache/brooklyn-library/blob/master/examples/simple-web-cluster/src/main/resources/visitors-creation-script.sql) of the visitors creation script. We then found out a lot of downstream projects were using it. Do you think it would be worth adding a [default](http://freemarker.org/docs/dgui_template_exp.html#dgui_template_exp_missing_default) here in case that is true for this file?